moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -48,6 +48,8 @@ export const signalToFeatureCollection = signal => {
       properties: {
         ...signal?.properties,
         renderType: "Point",
+        // I was considering changing this to be INTERSECTIONID, but since the sourceLayer is being saved
+        // as "drawnByUser" and not "ATD_ADMIN.CTN_Intersections", I've left it as PROJECT_EXTENT_ID
         PROJECT_EXTENT_ID: featureUUID,
         sourceLayer: "drawnByUser",
```

Comment:
see my note to the right -- I'll delete the comment once y'all have reviewed the code

Reply:
👍 looks right to me @chiaberry